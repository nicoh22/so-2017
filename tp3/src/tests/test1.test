load tests/test1.txt
maximum
quit
