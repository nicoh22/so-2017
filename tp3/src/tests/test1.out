[MPI] Lanzando proceso 4
Soy un nodo. Mi rank es 4 
[MPI] Lanzando proceso 0
Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> [MPI] Lanzando proceso 3
Soy un nodo. Mi rank es 3 
[MPI] Lanzando proceso 1
[MPI] Lanzando proceso 2
Soy un nodo. Mi rank es 2 
Soy un nodo. Mi rank es 1 
La lista esta procesada
> > El máximo es <casa,3>
> > El string <casa1> está
> El string <gato> está
> El string <perro> está
> El string <estoy> no está
> > El máximo es <casa,3>
> > Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> Agregado: casa2
> > El máximo es <casa,10>
> [mpiexec@Reimu] Sending Ctrl-C to processes as requested
[mpiexec@Reimu] Press Ctrl-C again to force abort
